go.mod

```diff
@@ -33,19 +33,19 @@ require (
 	github.com/stretchr/testify v1.9.0
 	github.com/summerwind/h2spec v0.0.0-20200804131034-70ac22940108
 	github.com/tcnksm/go-httpstat v0.2.1-0.20191008022543-e866bb274419
-	go.uber.org/zap v1.26.0
-	golang.org/x/time v0.3.0
+	go.uber.org/zap v1.27.0
+	golang.org/x/time v0.7.0
 	google.golang.org/api v0.126.0
 	google.golang.org/grpc v1.65.0
 	gopkg.in/fsnotify.v1 v1.4.7
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.31.1
-	k8s.io/apiextensions-apiserver v0.31.1
-	k8s.io/apimachinery v0.31.1
-	k8s.io/apiserver v0.31.1
+	k8s.io/api v0.32.1
+	k8s.io/apiextensions-apiserver v0.32.1
+	k8s.io/apimachinery v0.32.1
+	k8s.io/apiserver v0.32.1
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
-	sigs.k8s.io/controller-runtime v0.19.0
+	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
+	sigs.k8s.io/controller-runtime v0.20.0
```

Comment:
0.20.1 is out (that was quick). I see one [fix](https://github.com/kubernetes-sigs/controller-runtime/pull/3086) that might be of interest. Up to you if you want to bump again.

https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.20.1

Reply:
Right, I saw they dropped a new one. Let me add it.